deps/spidershim/test/v8v8.cc

```diff
@@ -0,0 +1,15 @@
+// Copyright 2015 the V8 project authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "v8engine.h"
+
+#include "gtest/gtest.h"
+
+TEST(SpiderShim, Version) {
+  v8::V8::GetVersion();
```

Comment:
Why not make this actually test something?  Like compare it against `"JavaScript-C<version>" or something like that using `strncmp()`?